seminar_registration_system/member_info_capture/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import MemberInfoCapture
+
+class MemberInfoCaptureSerializer(serializers.ModelSerializer):
```

Comment:
define `serialziers.py` file on my app to convert your django model to JSON  format